app/services/rbac_api.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# This class is meant to handle calls to the RBAC light API
+class RBACAPI
+  def initialize(account)
+    @url = URI.parse("#{URI.parse(Settings.remediations_url)}"\
```

Comment:
I think we need a new setting. In prod, this could look like: `remediations_url: http://remediations.remediations-ci.svc.cluster.local:9002`

Reply:
Good one - yup  indeed we needed it